scripts/testenv/testenvSettings.sh

```diff
@@ -1,18 +1,27 @@
 #!/usr/bin/env bash
 set +x
+set -e
 
-if [[ $USE_TESTENV_PROPERTIES == true ]]; then
-    testenv_file="./testenv/testenv.properties"
-    if [[ "$PLATFORM" == *"zos"* ]]; then
-        testenv_file="./testenv/testenv_zos.properties"
-    fi
-    if [[ "$PLATFORM" == *"arm"* ]] && [[ "$JDK_VERSION" == "8" ]]; then
-        testenv_file="./testenv/testenv_arm32.properties"
-    fi
+if [ $USE_TESTENV_PROPERTIES = true ]; then
+    case "$PLATFORM" in
+        *zos*)
+            testenv_file="./testenv/testenv_zos.properties"
+            ;;
+        *arm*)
+            if [ "$JDK_VERSION" = "8" ]; then
+                testenv_file="./testenv/testenv_arm32.properties"
+            else
+                testenv_file="./testenv/testenv.properties"
+            fi
+            ;;
+        *)
+            testenv_file="./testenv/testenv.properties"
```

Comment:
Rather than repeat the default, perhaps it should be set at the beginning like the previous code.